src/www/services_unbound_host_edit.php

```diff
@@ -58,9 +59,17 @@ function hosts_sort() {
 	$id = $_POST['id'];
 
 if (isset($id) && $a_hosts[$id]) {
+/* Backwards compatibility for records created before introducing different RR types. */
+	if (!isset($a_hosts[$id]['rr'])) {
+		$a_hosts[$id]['rr'] = 'A';
+	}
+
 	$pconfig['host'] = $a_hosts[$id]['host'];
 	$pconfig['domain'] = $a_hosts[$id]['domain'];
+	$pconfig['rr'] = $a_hosts[$id]['rr'];
 	$pconfig['ip'] = $a_hosts[$id]['ip'];
+	$pconfig['mxprio'] = $a_hosts[$id]['mxprio'];
+	$pconfig['mx'] = $a_hosts[$id]['mx'];
```

Comment:
The model is a bit too fragile for my taste: MX, like A or AAAA is a property of the domain itself and those entries can coexist, but here it is flat. What if we need multiple records for the same host? Is it wise to handle them "flat"?